### ISSUE TITLE
No restart of application (restartAfterDeploy=false) 

### ISSUE THREAD

**Issue by FJGL** [orctom/was-maven-plugin#11]
---

WASX7017E: Exception received while running file "...\was-maven-plugin\py\deploy-localhost-APP-8.1-SNAPSHOT-20151112-182325-345.py"; exception information: javax.management.MBeanException
javax.management.ServiceNotFoundException: Operation startApplication not in ModelMBeanInfo

It works changing to type=ApplicationManager instead of type=Server in websphere.py

```
       if "" != cluster:
            appManager = AdminControl.queryNames('type=ApplicationManager,process=' + server + ',*')
        elif "" != node:
            appManager = AdminControl.queryNames('node=' + node + ',type=ApplicationManager,process=' + server + ',*')
        else:
            appManager = AdminControl.queryNames('type=ApplicationManager,process=' + server + ',*')
```

Regards

**Comment by orctom** [comment]
Good catch! Thank!

**Comment by orctom** [comment]
Fixed in version `1.1.1`
